man/add_ws_to_parse_data.Rd

```diff
@@ -0,0 +1,28 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/parsed.R
+\name{add_ws_to_parse_data}
+\alias{add_ws_to_parse_data}
+\title{Pre-processing parse data}
+\usage{
+add_ws_to_parse_data(parse_data)
+}
+\arguments{
+\item{parse_data}{a parse table.}
+}
+\value{
+a pre-processed parse table.
+}
+\description{
+Pre-processing parse data
```

Comment:
The title is used as description if you omit the description in the documentation. The description can be more than one paragraph if you use the `@description` tag (I think...).